### PR TITLE
Load workflow & rancher source from an exteral config

### PR DESCRIPTION
**Pull request by bear454**
A yaml file injected into the config dir as `config.yml` will supply the workflow & rancher install source for _helm_. A well-documented example is included, while the config itself is explicitly excluded from source control and packaging.

### CODE REVIEW

**Comment by bear454** [comment]
Bonus: reduced the character count using `Rails.configuration` instead of `Rails.application.config` throughout _because my fingers are wearing out_.

**Review by jesusbv — APPROVED** [review]
LGTM

File: config/config.yml.example [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+shared:
+
+  # Which engines are loaded for this workflow?
+  # * Order determines menu order
+  # * Engines without UI components should be last
+  engines:
+    - AWS
+    - ShirtSize
+    - RancherOnEks
```
Comment (rjschwei) [inline config/config.yml.example:13]: Is the k8s version setting for the control plane targeted for a separate PR. Based on this factoring in config I could see a

```
- RancherOnEks:
  - k8s_version: 1.22
```

which can then be nicely carried forward to other CSPs, for example

```
- RancherOnAzure:
  - k8s_version: 1.23
```
Such a config would also help in accommodating version drift of available k8s versions across CSPs.

Comment (bear454) [reply config/config.yml.example:13]: Good call. I can do that as a follow-up

**Review by rjschwei — APPROVED** [review]

File: config/config.yml.example (hunk 1 above)
Comment (bear454) [reply config/config.yml.example:13]: I filed #102 for tracking.

**Pull request merged**